assets/js/components/module/ModuleApp.js

```diff
@@ -54,19 +51,7 @@ function ModuleApp( { moduleSlug } ) {
 
 	return (
 		<Fragment>
-			<Header>
-				<HelpMenu>
```

Comment:
@hussain-t the `HelpMenu` should not have been removed here, only the added item for AdSense as this is item is handled in the `HelpMenu` component now. cc: @eugene-manuilov